src/NServiceBus.Transport.Msmq/MsmqScopeOptions.cs

```diff
@@ -13,9 +12,7 @@ public MsmqScopeOptions(TimeSpan? requestedTimeout = null, IsolationLevel? reque
             var isolationLevel = IsolationLevel.ReadCommitted;
             if (requestedTimeout.HasValue)
             {
-                var maxTimeout = GetMaxTimeout();
-
-                if (requestedTimeout.Value > maxTimeout)
+                if (requestedTimeout.Value > TransactionManager.MaximumTimeout)
                 {
                     throw new ConfigurationErrorsException(
```

Comment:
Nit: Technically we could now replace this exception as well since we don't read anything from the config anymore. Probably a simple general exception or an ArgumentOutOfRangeException would be more descriptive here

Reply:
I would even say that previously it should also have been an argument exception as the value to compare is with is from the arguments.